v_4_0_0/files/k8s-resource/calico-all.yaml

```diff
@@ -215,7 +215,7 @@ spec:
               memory: 150Mi
             limits:
               cpu: 250m
-              memory: 150Mi
+              memory: 250Mi
```

Comment:
This is ~67% increase should be plenty enough. Normally it only seldom goes above `150MB`, though I admit that prometheus scrape interval of 30s doesn't help with this. Nevertheless, so far manual adjustment of memory limit to `250MB` has removed the problem so hence I'm rather confident with this. The downside of having too large limit is that it prevents usage of small instance types because they cannot hold all workload. This overhead is then also present on every node.

Following screenshot is past week from `asgard`:
![Screenshot from 2019-09-27 12-19-30](https://user-images.githubusercontent.com/1947505/65758152-3b812300-e121-11e9-8088-490150e04403.png)